sass/directives/_flyout.scss

```diff
@@ -87,3 +87,117 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+// Structure of markup should be:
+@mixin mobile-subnav-flyout() {
+  width: auto; 
+  display: none;
+  position: relative;
+  cursor: pointer;
+
+  @media only screen and (max-width: $small-screen-max) {
+    line-height: 45px;
```

Comment:
This was removed due to the refactor, but instances that are based off the height of the subnav itself (45px) are now using this variable.